frontend/src/components/navigations/top-nav-bar/TopNavBar.jsx

```diff
@@ -68,27 +68,23 @@ function TopNavBar() {
         !isUnstractSupervisor
     );
 
-    setApproverStatus(
-      (isUnstractAdmin || isUnstractSupervisor) && flags.manual_review
-    );
-    setReviewerStatus(isUnstractReviewer && flags.manual_review);
+    setApproverStatus(isUnstractAdmin || isUnstractSupervisor);
+    setReviewerStatus(isUnstractReviewer);
   }, [sessionDetails]);
 
   useEffect(() => {
-    if (flags.manual_review) {
-      const checkReviewPage = location.pathname.split("review");
+    const checkReviewPage = location.pathname.split("review");
 
-      if (checkReviewPage.length > 1) {
-        if (checkReviewPage[1].includes("/approve")) {
-          setReviewPageHeader("Approve");
-        } else if (checkReviewPage[1].includes("/download_and_sync")) {
-          setReviewPageHeader("Download and syncmanager");
-        } else {
-          setReviewPageHeader("Review");
-        }
+    if (checkReviewPage.length > 1) {
+      if (checkReviewPage[1].includes("/approve")) {
+        setReviewPageHeader("Approve");
+      } else if (checkReviewPage[1].includes("/download_and_sync")) {
+        setReviewPageHeader("Download and syncmanager");
       } else {
-        setReviewPageHeader(null);
+        setReviewPageHeader("Review");
       }
+    } else {
+      setReviewPageHeader(null);
     }
```

Comment:
@vishnuszipstack NIT: I think this useEffect block can be further modified to improve code readability. Please ignore if you think otherwise.

```
  const checkReviewPage = location.pathname.split("review");

  if (checkReviewPage.length <= 1) {
    setReviewPageHeader(null);
    return;
  }

  if (checkReviewPage[1].includes("/approve")) {
    setReviewPageHeader("Approve");
    return;
  }
  
  if (checkReviewPage[1].includes("/download_and_sync")) {
    setReviewPageHeader("Download and syncmanager");
    return;
  }
  
  setReviewPageHeader("Review");
```

Reply:
Will do this in V2 PR @tahierhussain . This PR only meant to remove the feature flag.